3주차/styles/index_style.css

```
@@ -0,0 +1,385 @@
+  position: relative;
+  background-image: url('../images/page_2.png');
+  background-blend-mode: multiply;
+  background-position: center;
+  background-size: 160%;
+  overflow: hidden;
+}
+
+.background > img {
+  position : absolute;
+  top : 12.1rem;
+  left: 7.7rem;
+  width: 41.6rem;
+}
+
+.search-result {
+  position : absolute;
+  top : 5.5rem;
+  left: 6rem;
+  width: 61.9rem;
+  border-radius: 15px;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  width : 100%;
+  height: 16rem;
+  padding-top: 3.2rem;
+  background-color: #111322;
+}
+
+.footer-box {
+  height : fit-content;
+  background-color: #111322;
+  padding: 0 10.4rem;
+  display: flex;
+  justify-content: space-between;
+  width : 100%;
+}
+
+.footer-links {
+  display: flex;
+  column-gap: 3rem;
+  padding-right: 1.8rem;
+}
+
+.footer-link {
+  color: #cfcfcf;
+  font-family: 'Arial';
+  font-size: 1.6rem;
+  text-decoration: none;
+}
+
+.icons {
+  display: flex;
+  column-gap: 1.2rem;
+  height: 2remx;
+}
+
+
+
+/* 테블릿 화면에 적용 */
+@media (max-width: 1199px){
+  nav{
+    padding : 2rem 3.2rem;
+  }
+  nav > div {
+    min-width : 703px;
+  }
+  .blue-header{
+    width : 69.8rem;
+    height: 71.7rem;
+    padding-top: 3.9rem;
+  }
+
+  .all-info{
+    width : 47.2rem;
+  } 
+  .all-info > span {
+    letter-spacing: -0.1rem;;
+  }
+
+  .header-image-box {
+    width: 65rem;
+  }
+  .header-image{
+    width : 100%;
+  }
+
+  .large-article{
+    width : 32.5rem;
+    height: 26.5rem;
+  }
+  .images > img {
+    width : 19rem;
+    height: 19rem;
+  }
+  .name-change {
+    width: 28.2rem;
+    height: 18.9rem;
+  }
+  .background {
+    width: 32.5rem;
+    height: 26.5rem;
+  }
+  .background > img {
+    width : 29rem;
+    height: 18.9rem;
+    top : 5rem;
+    left : 2.5rem;
+  }
+  .search-result{
+    width: 42.4rem;
+    top : 4rem;
+    left: 5rem;
+    width: 110%;
+  }
+ 
+}
+
+
+
+/* 모바일 화면에 적용 */
+@media (max-width : 766px){
+  
+  nav{
+    padding : 13px 32px;
+  }
+  nav > div {
+    min-width : 326px;
+  }
+
+  .title{
+    width : 8.8rem;
+  }
+
+  .button {
+    font-size : 14px;
+  }
+
+  .login{
+    width: 8rem;
+  }
+
+  .blue-header{
+    gap : 2.4rem;
+    width : 100%;
+    height: 39.9rem;
+    padding-top: 2.8rem;
+    padding-left: 3.3rem;
+    padding-right: 3.2rem;
+  }
+
+  .all-info{
+    width : 23.6rem;
+    font-size : 3.2rem;
+    line-height: 4.2rem;
+  } 
+  .all-info > span {
+    font-size : 3.2rem;
+    line-height: 4.2rem;
+    letter-spacing: -0.1rem;;
+  }
+
+  .header-image-box {
+    width: 100%;
+  }
+  .large-contents{
+    padding : 0;
+  }
+
+  .content{
+    display: flex;
+    flex-direction: column;
+    gap: 2.4rem;
+    padding: 4rem 3.2rem;
+  }
+
+  .large-article{
+    width : 100%;
+  }
+
+  .images > img {
+    width : 50%;
+  }
+ 
+  .background {
+    width: 100%;
+  }
+  .background > img {
+    left: 0;
+  }
+  .search-result{
+    width: 42.4rem;
+    top : 5rem;
+    left: 4.8rem;
+  }
```

항상 아래와 같이 일관성 있는 포맷팅이 중요합니다! 👍 사소하지만 꼭 챙겨주시거나 formatting extension 의 도움을 받아보시길 바랍니다!

```suggestion
  .background > img {
    left: 0;
  }
  
  .search-result {
    width: 42.4rem;
    top : 5rem;
    left: 4.8rem;
  }
```